scrapi/base/schemas.py

```diff
@@ -7,18 +7,21 @@
     oai_extract_url,
     # oai_extract_doi,
     oai_process_contributors,
+    compose,
+    single_result,
     format_tags
 )
 
 CONSTANT = lambda x: lambda *_, **__: x
 
+
 BASEXMLSCHEMA = {
-    "description": ('//dc:description/node()', lambda x: unicode(x.strip())),
-    "contributors": ('//dc:creator/node()', lambda x: default_name_parser(x.split(';'))),
-    "title": ('//dc:title/node()', lambda x: unicode(x.strip())),
-    "providerUpdatedDateTime": ('//dc:dateEntry/node()', lambda x: unicode(x.strip())),
+    "description": ('//dc:description/node()', compose(lambda x: unicode(x.strip()), single_result)),
```

Comment:
That won't play nicely with unicode/python 3, will it? Also, is there actually any necessity that we cast to unicode anymore?